job-controller/src/main/java/feast/jobcontroller/config/JobControllerConfig.java

```diff
@@ -87,9 +87,9 @@ public JobGroupingStrategy getJobGroupingStrategy(
     Boolean shouldConsolidateJobs =
         feastProperties.getJobs().getController().getConsolidateJobsPerSource();
     if (shouldConsolidateJobs) {
-      return new ConsolidatedJobStrategy(jobRepository);
+      return new ConsolidatedJobStrategy(jobRepository, feastProperties.getJobs());
```

Comment:
Can we extract the jobProperties before passing it? It's a convention that I am trying to get us to follow, even if it takes more lines of code

Reply:
Extracted before passing jobProperties.